monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -495,9 +495,18 @@ abstract class Observable[+A] extends Serializable { self =>
     self.liftByOperator(new CollectOperator(pf))
 
   /** Creates a new observable from the source and another given
-    * observable, by emitting elements combined in pairs. If one of
-    * the observables emits fewer events than the other, then the rest
-    * of the unpaired events are ignored.
+    * observable, by emitting elements combined in pairs.
+    *
+    * It emits an item whenever any of the source Observables emits an
+    * item (so long as each of the source Observables has emitted at
+    * least one item).
+    *
+    * Example:
+    *
+    * stream1: 1 - - 2 - - 3 - 4 - -
+    * stream2: 1 - - 2 - 3 - - - - 4
```

Comment:
@Avasil sorry for not reviewing this earlier.

This is really cool, however I don't think it will get formatted properly when ScalaDoc is generated, because newlines are not taken into account.

Reply:
In this case I'll try generating scaladoc and see if I can make it wiek / close PR otherwise

Reply:
Go for it. You probably need to introduce some sort of `<pre>` block.

Reply:
@alexandru `<pre>` tags were enough and IMO it looks quite nice:

![image](https://user-images.githubusercontent.com/12383899/46064674-59175780-c170-11e8-8496-ccd84d86dbfa.png)

The only issue is extra space but it's not too bad

Reply:
👍 looks good